payments-core/src/main/java/com/stripe/android/model/ConfirmSetupIntentParams.kt

```diff
@@ -38,6 +38,13 @@ constructor(
 
     private val useStripeSdk: Boolean = false,
 
+    /**
+     * Payment-method-specific configuration for this SetupIntent.
+     *
+     * See [payment_method_options](https://docs.stripe.com/api/setup_intents/confirm#confirm_setup_intent-payment_method_options).
+     */
+    var paymentMethodOptions: PaymentMethodOptionsParams? = null,
```

Comment:
Can this be `val` instead?
```suggestion
    val paymentMethodOptions: PaymentMethodOptionsParams? = null,
```